extras/centos_bootstrap.sh

```diff
@@ -21,14 +21,24 @@ yum update -y && yum upgrade -y
 ### Install the YETI Dependencies
 yum groupinstall "Development Tools" -y
 yum install epel-release
-yum install python-pip git mongodb-org python-devel libxml2-devel libxslt-devel zlib-devel redis firewalld yarn -y
+yum install python-pip git mongodb-org python-devel libxml2-devel libxslt-devel zlib-devel redis firewalld yarn vim curl wget net-tools nginx uwsgi -y 
 pip install --upgrade pip
+pip install django libtaxii pyOpenSSL uwsgi cabby django-solo taxii_services python-dateutil
```

Comment:
Why are you installing `django`?

Reply:
good catch it looks like the install requirements from the TAXIIProject yeti project that I was looking at got added here by mistake. I'm cleaning that up now. it should only be uwsgi.